File: springBootServer/simulator-api/cpp/PhysicEngine/ConstantsCarrom.cpp
#include "ConstantsCarrom.h"
#include <cmath>

const double ConstantsCarrom::mu = 0.18;
const double ConstantsCarrom::mur = 0.085;
const double ConstantsCarrom::g = 9.80665;
const double ConstantsCarrom::rho = 0.002;
const double ConstantsCarrom::m = 0.018;
const double ConstantsCarrom::R = 0.0202565;
const double ConstantsCarrom::Mz = ((mu * m * g * 2) / 3) * rho;
const double ConstantsCarrom::Mxy = (7.0 / (5 * std::sqrt(2))) * R * mur * m * g;
const double ConstantsCarrom::e = 0.97;
const double ConstantsCarrom::I = (2.0 / 5.0) * m * R * R;

File: springBootServer/simulator-api/src/main/java/com/alfa/carromclash/GameEngine/Core/Physic/Engine/V1/ConstantsCarrom.java
package com.alfa.carromclash.GameEngine.Core.Physic.Engine.V1;

public class ConstantsCarrom {
    public static double mu = 0.18;
    public static double mur = 0.085;
    public static double g =  9.80665; //9.80
    public static double rho = 0.002;
    public static double m = 0.018;
    public static double R = 0.0202565;
    public static double Mz = ((mu * m * g * 2) / 3) * rho;
    public static double Mxy = (7d / (5 * Math.sqrt(2))) * R * mur * m * g;
    public static double e = 0.97d;
    public static double I = (2d / 5d) * m * R * R;

}
